resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.init.erb

```diff
@@ -62,6 +64,11 @@ do_start()
     <%= action %>
     <% end -%>
 
+    if [ ! -d "/var/run/puppetlabs/${REALNAME}" ] ; then
```

Comment:
I believe `install` would just effectively do nothing if the directory already exists with the intended permissions.  Only reason I'm throwing that out is that we've been doing it this way via the pre-start-action in Puppet Server for a number of releases without any problems and it does seem a little more robust for the case that the directory might already exist with the wrong permissions at start time.  I could go either way on that, though.  @haus - do you have a strong opinion on this?